apps/backend/src/modules/projects/project.controller.ts

```diff
@@ -129,6 +143,41 @@ class ProjectController extends BaseController {
 			status: HTTPCode.OK,
 		};
 	}
+
+	/**
+	 * @swagger
+	 * /projects/{id}:
+	 *    get:
+	 *      description: Get project by ID
+	 *      parameters:
+	 *        - in: path
+	 *          name: id
+	 *          schema:
+	 *            type: integer
+	 *          required: true
+	 *          description: Numeric ID of the project to retrieve
+	 *      responses:
+	 *        200:
+	 *          description: Successful operation
+	 *          content:
+	 *            application/json:
+	 *              schema:
+	 *                type: object
+	 *                properties:
+	 *                  message:
+	 *                    type: object
+	 *                    $ref: "#/components/schemas/Project"
+	 */
+	private async getById(
+		options: APIHandlerOptions<{
+			params: ProjectGetByIdRequestDto;
```

Comment:
let's remove ProjectGetByIdRequestDto, request dto types is only for body payload, not for path parameters. We define parameters explicitly
```suggestion
			params: { id: string };
```